extensions/wikia/ArticleVideo/scripts/recommended-video.jwplayer.js

```diff
@@ -0,0 +1,210 @@
+require([
+	'jquery',
+	'wikia.tracker',
+	'wikia.articleVideo.featuredVideo.cookies',
+	'wikia.loader',
+	'wikia.mustache'
+], function ($, tracker, featuredVideoCookieService, loader, mustache) {
+	'use strict';
+
+	var jwPlaylistDataUrl = 'https://cdn.jwplayer.com/v2/playlists/',
+		$unit = $('.article-recommended-video-unit'),
+		scrollBreakpoint = 100,
+		recommendedVideoElementId = 'recommendedVideo',
+		recommendedVideoData = null,
+		player = null,
+		$actualVideo = null,
+		currentItemNumber = 1,
+		isExpanded = false;
+
+	function reveal() {
+		var scrollTop = $(window).scrollTop();
+
+		if (scrollTop > scrollBreakpoint) {
+			$unit.addClass('is-revealed');
+			window.wikiaJWPlayer(
+				recommendedVideoElementId,
+				getPlayerSetup(recommendedVideoData),
+				onPlayerReady
+			);
+
+			tracker.track({
+				category: 'recommended-video',
+				trackingMethod: 'both',
+				action: tracker.ACTIONS.VIEW,
+				label: 'recommended-video-revealed'
+			});
+		}
+	}
+
+	function onScroll() {
+		var scrollTop = $(window).scrollTop();
+
+		if (scrollTop > scrollBreakpoint) {
+			window.setTimeout(reveal, 5000);
+		}
+	}
+
+	function loadAssets() {
+		loader({
+			type: loader.MULTI,
+			resources: {
+				mustache: 'extensions/wikia/ArticleVideo/templates/ArticleVideo_recommendedVideos.mustache'
+			}
+		}).done(function (assets) {
+			renderView(assets.mustache[0]);
+			window.document.addEventListener('scroll', onScroll);
+		});
+	}
+
+	function renderView(view) {
+		// This is needed in order to access item index in mustache
+		recommendedVideoData.playlist.forEach(function (video, index) {
+			video.index = index;
+		});
+		$unit.find('.article-recommended-videos').html(
+			mustache.render(view, {
+				playlist: recommendedVideoData.playlist,
+				firstVideo: recommendedVideoData.playlist[0]
+			})
+		);
+
+		$actualVideo = $unit.find('.actual-video');
```

Comment:
what is actual? current?

Reply:
Actual is a container of the jwplayer. All the others are just placeholders.